src/components/Header/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PureComponent } from 'react';
+import './styles.scss';
+
+class Header extends PureComponent {
+  render() {
+    return (
+      <div id="header">
+        <div className="brand">
+          {/* eslint-disable jsx-a11y/anchor-is-valid */}
+          <a href="#">
+            <img className="logo" src="/logo.png" alt="Andela Logo" />
+            <span className="text">ESA Reporting</span>
+          </a>
+        </div>
+        <div className="sign-out">
+          <a href="#" className="sign-out">
+            <i className="fa fa-sign-out-alt" />&nbsp;
+            <span>Sign Out</span>
+          </a>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
I think we need that user icon back.

Reply:
Hey @Betty-Kebenei
I removed the user icon because I did not see any reason to have it there. But I would like to know if there is a reason to have it there.

Reply:
Hmm UX...
Speaking about UX, the only thing that the user can do on this app is simply to get authenticated(to login) and log out.
If we are going to add a user icon, then the question would be what would the user icon be used for?

Reply:
Allow me to ask; what is the user icon in all these other sites?

Reply:
This conversation was resolved in a sync with the TTL for this project(@tadesanya).
The resolution was to add the user-icon to the header such that after the user clicks the icon, the signout link is displayed in a drop-down.
I have updated the codes to reflect that implementation cc: @Betty-Kebenei